src/configeditor/EditBoundingBoxes.tsx

```diff
@@ -0,0 +1,208 @@
+}
+
+interface EditSingleBoundingBoxesProps {
+  type: string;
+  bounds: BoundingBox | null;
+  values: any;
+  setValues: any;
+  error: boolean;
+}
+
+function LatLngInput(props: LatLngInputProps) {
+  return (
+    <FormControl isInvalid={props.error}>
+      <FormLabel htmlFor={props.field}>{props.title}</FormLabel>
+      <Input
+        id={props.field}
+        type="number"
+        variant="outline"
+        value={props.value}
+        onChange={props.onChange}
+      />
+    </FormControl>
+  )
+}
+
+function EditSingleBoundingBoxes(props: EditSingleBoundingBoxesProps) {
+  const { type, bounds, values, setValues, error } = props;
+
+  const onChange = (value: string, field: "westmost" | "southmost" | "eastmost" | "northmost") => {
+    if (!bounds) {
+      const newBounds = (
+        field === "westmost" ? new BoundingBox(value, "", "", "") :
+        field === "southmost" ? new BoundingBox("", value, "", "") :
+        field === "eastmost" ? new BoundingBox("", "", value, "") :
+        new BoundingBox("", "", "", value)
+      );
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: newBounds.toConfigBbox()
+        }
+      });
+    } else {
+      bounds[field] = value;
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: bounds.toConfigBbox()
+        }
+      });
+    };
+  };
+
+  return (
+    <HStack align="start">
+      <LatLngInput
+        title="West"
+        field="westmost"
+        value={bounds ? bounds.westmost : ""}
+        onChange={e => onChange(e.target.value, "westmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="South"
+        field="southmost"
+        value={bounds ? bounds.southmost : ""}
+        onChange={e => onChange(e.target.value, "southmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="East"
+        field="eastmost"
+        value={bounds ? bounds.eastmost : ""}
+        onChange={e => onChange(e.target.value, "eastmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="North"
+        field="northmost"
+        value={bounds ? bounds.northmost : ""}
+        onChange={e => onChange(e.target.value, "northmost")}
+        error={error}
+      />
+      <button
+        key={type}
+        title={"Clear input"}
+        onClick={() => setValues({
+          ...values,
+          boundingBoxes: {
+            ...values.boundingBoxes,
+            [type]: null
+          }
+        })}
+      >
+        &#x2715;
+      </button>
+    </HStack>
+  )
+}
+
+function EditBoundingBoxes() {
+  const { status, values, setValues, errors, submit } = useConfigEdit(["boundingBoxes"]);
+
+  // bounding boxes of different types of map
+  const boundingBoxes = values.boundingBoxes as Config["boundingBoxes"];
+  const defaultBounds = boundingBoxes.default ? new BoundingBox(...boundingBoxes.default) : null;
```

Comment:
When the default values PR is merged, these can take defaults from constants/DEFAULT_CONFIG (which currently has an extent for Parramatta as default).

Reply:
changed to use from DEFAULT_CONFIG.boundingboxes.default instead of using null for defaultBounds.

Reply:
I made this commit: https://github.com/nens/flood-early-warning-interface/pull/68/commits/ecad75b5584627521f2f9db855cb4517af0ad818 to run an useEffect only when component first mounted with no default bounding box value. It will call `setValues` to set the default bounding box to the value in the DEFAULT_CONFIG.